traitcuration/traits/utils.py

```diff
@@ -38,6 +39,55 @@ def parse_request_body(request):
     return body
 
 
+def add_ontology_sources_to_context(context):
+    ontology_sources = list()
+
+    latest_ols_import = TaskResult.objects.filter(task_name="traitcuration.traits.tasks.import_ols").first()
+    if latest_ols_import:
+        latest_import_date = latest_ols_import.date_done
+        ols_task_id = latest_ols_import.task_id
+    else:
+        latest_import_date = None
+        ols_task_id = "None"
+    context['ols_task_id'] = ols_task_id
+    ols_dict = {'id': 'ols', 'title': 'Source of ontology term information',
+                'latest_import_date': latest_import_date}
+    ontology_sources.append(ols_dict)
+
+    latest_zooma_import = TaskResult.objects.filter(
+        task_name="traitcuration.traits.tasks.import_zooma").first()
+    if latest_zooma_import:
+        latest_import_date = latest_zooma_import.date_done
+        zooma_task_id = latest_zooma_import.task_id
+    else:
+        latest_import_date = None
+        zooma_task_id = "None"
+    context['zooma_task_id'] = zooma_task_id
+    zooma_dict = {'id': 'zooma', 'title': 'Source of mapping suggestions',
+                  'latest_import_date': latest_import_date}
+    ontology_sources.append(zooma_dict)
+
+    context['ontology_sources'] = ontology_sources
+
+
+def add_trait_sources_to_context(context):
+    trait_sources = list()
+
+    latest_clinvar_import = TaskResult.objects.filter(task_name="traitcuration.traits.tasks.import_clinvar").first()
+    if latest_clinvar_import:
+        latest_import_date = latest_clinvar_import.date_done
+        clinvar_task_id = latest_clinvar_import.task_id
+    else:
+        latest_import_date = None
+        clinvar_task_id = "None"
+    context['clinvar_task_id'] = clinvar_task_id
+    clinvar_dict = {'id': 'clinvar', 'title': 'ClinVar',
+                    'latest_import_date': latest_import_date}
+    trait_sources.append(clinvar_dict)
+
+    context['trait_sources'] = trait_sources
```

Comment:
Same comment about (mild) code block duplication applies here. Right now it's probably not worth the effort to refactor that, but once we have more sources, it will become necessary

Reply:
As with the above, it is kinda tricky to condense that, I will rethink that, but I think we should merge this PR and add a follow up issue about that.

Reply:
Agreed